src/main/java/Lemon.java

```diff
@@ -0,0 +1,135 @@
+import java.util.Scanner;
+
+public class Lemon {
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+
+        System.out.println("——————————————————————————————————————————————");
+        System.out.println("Hello! I'm Lemon");
+        System.out.println("What can I do for you?");
+        System.out.println("——————————————————————————————————————————————");
+
+        Scanner scanner = new Scanner(System.in);
+        Task[] tasks = new Task[100];
+        int index = 0;
+
+        while(true) {
```

Comment:
```suggestion
        while (true) {
```
You can consider leaving a space after each if / while / for etc :)